docs/copilot/getting-started.md

```diff
@@ -1,15 +1,15 @@
 ---
-Order: 10
+Order: 3
 Area: copilot
-TOCTitle: Getting started tutorial
+TOCTitle: Getting Started Tutorial
 ContentId: 37fd3bd2-4209-49f6-bec5-c544d6b1b289
 PageTitle: GitHub Copilot getting started
 DateApproved: 04/04/2024
 MetaDescription: Get started with GitHub Copilot in Visual Studio Code and create your first AI-powered suggestions in the editor.
 ---
 # Getting started with GitHub Copilot in VS Code
 
-This tutorial walks you through setting up the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension, and discovering the key features of GitHub Copilot in Visual Studio Code. Learn how to get started with AI-powered code suggestions, use chat conversations to refactor your code, and fix code errors with smart actions.
+This tutorial walks you through using the [GitHub Copilot](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) extension to discover the key features of GitHub Copilot in Visual Studio Code. Learn how to get started with AI-powered code suggestions, use chat conversations to refactor your code, and fix code errors with smart actions.
```

Comment:
This stuck out to me a bit - are users really using Copilot to learn about Copilot?

Reply:
Since it's a tutorial, they learn hands-on about Copilot by using Copilot. I'll reword that part with the next Copilot updates.